common/lib/dependabot/ecosystem.rb

```diff
@@ -0,0 +1,171 @@
+# typed: strong
+# frozen_string_literal: true
+
+require "sorbet-runtime"
+
+module Dependabot
+  class Ecosystem
+    extend T::Sig
+
+    class VersionManager
```

Comment:
Previously, this class used an abstract `PackageManager` from `common/lib/dependabot/package_manager`, which has since been removed and replaced with an `Ecosystem` wrapper. The file is named `VersionManager` because we plan to introduce a `language` variable within `VersionManager`, which will provide the same stored inputs and functionality for the language used, similar to how it handles package manager information.